### ISSUE TITLE
JRE 1.8?

### ISSUE THREAD

**Issue by bhoeferlin** [KilianB/JImageHash#52]
Hi,

would it be possible to build (and deploy to maven) with Java 1.8?

**Comment by KilianB** [comment]
Yes this shouldn't be an issue. The only thing I have to investigate is how the javafx dependencies change. As far as I remember they were still included in 1.8?

**Comment by bhoeferlin** [comment]
Would really appreciate the 1.8 deployment - tnx!

**Comment by lightyear5235** [comment]
That seems cant be run on a JDK8 version project. Since the JImageHash is build by JDK11, which is setting in POM.xml.

**Comment by dendi875** [comment]
JDK8 uses version 2.1.1

**Comment by ferrymo** [comment]
> JDK8 uses version 2.1.1

openjdk?
